src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -1470,7 +1470,7 @@ namespace TerminalAppLocalTests
         CascadiaSettings settings{};
         settings._ParseJsonString(settingsJson, false);
         settings.LayerJson(settings._userSettings);
-        VERIFY_IS_FALSE(settings._profiles.empty(), 0);
+        VERIFY_IS_FALSE(settings._profiles.empty(), nullptr);

```

Comment:
These two look like they actaully don't need the param at all, and were mistakenly left from a previous iteration that was something like `VERIFY_ARE_EQUAL(.size(), 0)`

```suggestion
        VERIFY_IS_FALSE(settings._profiles.empty());
```